R/body.R

```diff
@@ -38,8 +37,7 @@ add_body <-
            row_style_names = NULL,
            left_header_style_names = NULL,
            col_style_names = NULL,
-           fill_na_with = NULL,
-           fill_nan_with = NULL) {
+           fill_non_values_with = list(na = NULL, nan = NULL, inf = NULL, neg_inf = NULL)) {
```

Comment:
what would happen if user passes e.g. `list(na = NULL)`  (i.e. doesn't pass all value).  Does this still work?

Reply:
Yup